calico_node/Dockerfile

```diff
@@ -14,29 +14,23 @@
 FROM gliderlabs/alpine:3.4
 MAINTAINER Tom Denham <tom@projectcalico.org>
 
-ENV GLIBC_VERSION 2.23-r1
 
-# Download and install glibc for use by the startup script
-RUN apk add --update curl && \
-  curl -o glibc.apk -L "https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk" && \
-  apk add --allow-untrusted glibc.apk && \
-  curl -o glibc-bin.apk -L "https://github.com/andyshinn/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-bin-${GLIBC_VERSION}.apk" && \
-  apk add --allow-untrusted glibc-bin.apk && \
-  /usr/glibc-compat/sbin/ldconfig /lib /usr/glibc/usr/lib && \
-  echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
-  apk del curl && \
-  rm -f glibc.apk glibc-bin.apk && \
-  rm -rf /var/cache/apk/*
+RUN apk --no-cache add wget ca-certificates libgcc && \
+    wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-2.23-r3.apk && \
+    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.23-r3/glibc-bin-2.23-r3.apk && \
+    apk add glibc-2.23-r3.apk glibc-bin-2.23-r3.apk && \
+    /usr/glibc-compat/sbin/ldconfig /lib /usr/glibc/usr/lib && \
+    apk del wget ca-certificates && \
+    rm -f glibc-2.23-r3.apk glibc-bin-2.23-r3.apk
```

Comment:
An apk is like a deb. We download it, then install it. There's no need to leave the file lying around wasting space.